lte/gateway/c/core/oai/tasks/ngap/CMakeLists.txt

```diff
@@ -16,16 +16,24 @@ set(ngap_generate_code_done_flag ${GENERATED_FULL_DIR}/NGAP_GENERATE_CODE_DONE)
 file(MAKE_DIRECTORY ${GENERATED_FULL_DIR})
 set(ngap_generate_code_done_flag ${GENERATED_FULL_DIR}/NGAP_GENERATE_CODE_DONE)
 
+# TODO: GH13021 this is a mitigation for excessive logging that should be handled more clean
+set(ASN1_ERR_FILTER "2> >(grep -v \"Parameterized type\" | grep -v \"Compiled \" | grep -v \"Copied \" >&2)")
+
 if( ${ASN1_SOURCE_DIR} IS_NEWER_THAN ${ngap_generate_code_done_flag})
    file(REMOVE ${GENERATED_FULL_DIR}/${ASN1C_PREFIX}*.c ${GENERATED_FULL_DIR}/${ASN1C_PREFIX}*.h)
    file(MAKE_DIRECTORY ${GENERATED_FULL_DIR})
 
-#Create NGAP_PDU structure without pointer references
- execute_process(COMMAND asn1c -pdu=all -fcompound-names -fno-include-deps -gen-PER -D ${GENERATED_FULL_DIR} ${ASN1_SOURCE_DIR} RESULT_VARIABLE ret)
-   if (NOT ${ret} STREQUAL 0)
-      message(FATAL_ERROR "${ret}")
-   endif (NOT ${ret} STREQUAL 0)
-   execute_process(COMMAND bash "-c" "egrep -lRZ \"18446744073709551615\" ${GENERATED_FULL_DIR} | xargs -0 -l sed -i -e \"s/18446744073709551615/18446744073709551615u/g\"")
+    #Create NGAP_PDU structure without pointer references
+    execute_process(
+        COMMAND bash "-c" "asn1c -pdu=all -fcompound-names -fno-include-deps -gen-PER -D ${GENERATED_FULL_DIR} ${ASN1_SOURCE_DIR} ${ASN1_ERR_FILTER}"
+        RESULT_VARIABLE ret
+    )
+    if (NOT ${ret} STREQUAL 0)
+        message(FATAL_ERROR "${ret}")
+    endif (NOT ${ret} STREQUAL 0)
+    execute_process(
+        COMMAND bash "-c" "egrep -lRZ \"18446744073709551615\" ${GENERATED_FULL_DIR} | xargs -0 -l sed -i -e \"s/18446744073709551615/18446744073709551615u/g\""
+    )
```

Comment:
The diff would be easier to read if the lines that don't change aren't reformatted.

Reply:
Would it help if I split this in two commits? 1. reformat, 2. add change.

Reply:
I would keep the reformatting at a minumum (besides making the diff in the PR unnecessarily long it also makes `git blame` less useful). As long as we don't have an autoformatter or very explicit formatting guidelines the ideal formatting remains subjective and I would only change existing code when it's really hard to read. I'd say this doesn't apply here. That said, yes if you do a reformatting + semantic changes, then I would say that splitting the reformatting out does make it easier to review.